pkg/commands/plan/plan.go

```diff
@@ -322,6 +325,30 @@ func (c *PlanCommand) terraformPlan(ctx context.Context) (*RunResult, error) {
 	stdout.Reset()
 	stderr.Reset()
 
+	if !c.skipJSONPlanFile {
+		var jsonOut strings.Builder
+		util.Headerf(c.Stdout(), "Writing Plan to Local JSON File")
+		if _, err = c.terraformClient.Show(ctx, &jsonOut, multiStderr, &terraform.ShowOptions{
+			File:    pointer.To(c.planFilename),
+			NoColor: pointer.To(true),
+			JSON:    pointer.To(true),
+		}); err != nil {
+			return &RunResult{
+				commentDetails: stderr.String(),
+				hasChanges:     hasChanges,
+			}, fmt.Errorf("failed to terraform show: %w", err)
+		}
+
+		planBasename := strings.TrimSuffix(c.planFilename, path.Ext(c.planFilename))
+		jsonFilepath := path.Join(c.childPath, planBasename+".json")
```

Comment:
We should figure out a better way for this, this feels hacky, but we can address in a follow up.

Reply:
I changed this to mimick the `planFilename` pattern for now